packages/wdio-cli/src/run.js

```diff
@@ -64,9 +64,24 @@ export default function run (params) {
     })
 }
 
+/**
+ * launcher callback function
+ * @param {number|undefined}    exitCode    exit code
+ * @param {Error|undefined}     err         Error
+ * @param {boolean|undefined}   isComplete  if all tests and hooks were executed
+ */
+const launchCallback = (exitCode, err, isComplete) => {
+    if (!err) {
+        return { exitCode }
+    }
+    console.error(err)
```

Comment:
**IMPORTANT**:
If error was thrown we are not printing out reporters!!!  I think this behavior is invalid! 
I can change it, if anyone find it useful.

Reply:
In my opinion it would be better to do in separate PR.

Reply:
agree